<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DayReserve</class>
 <widget class="QDialog" name="DayReserve">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>536</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../../icons/resources.qrc">
    <normaloff>:/png/marca-m.png</normaloff>:/png/marca-m.png</iconset>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>511</width>
     <height>541</height>
    </rect>
   </property>
   <property name="title">
    <string>Itens</string>
   </property>
   <widget class="QTableView" name="tableView">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>491</width>
      <height>510</height>
     </rect>
    </property>
    <property name="selectionBehavior">
     <enum>QAbstractItemView::SelectRows</enum>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>284</x>
     <y>566</y>
     <width>239</width>
     <height>25</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="pushButtonCancelarReserva">
      <property name="text">
       <string>Remover</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButtonEdit">
      <property name="text">
       <string>Editar</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButtonNew">
      <property name="text">
       <string>Nova</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <tabstops>
  <tabstop>pushButtonCancelarReserva</tabstop>
  <tabstop>pushButtonEdit</tabstop>
  <tabstop>pushButtonNew</tabstop>
  <tabstop>tableView</tabstop>
 </tabstops>
 <resources>
  <include location="../../icons/resources.qrc"/>
  <include location="../../icons/resources.qrc"/>
  <include location="../../icons/resources.qrc"/>
 </resources>
 <connections/>
</ui>
